src/transformStateWithClones.js

```diff
@@ -7,7 +7,40 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const array = [];
+  const obj = { ...state };
+
```

Comment:
const ADD = 'addProperties';

Consider using constants instead of magic strings here